frontends/p4/fromv1.0/programStructure.cpp

```diff
@@ -1385,7 +1653,7 @@ ProgramStructure::convertAction(const IR::ActionFunction* action, cstring newNam
 
     for (auto p : action->action) {
         auto stat = convertPrimitive(p);
-        if (stat != nullptr)
+        if (stat != nullptr /* && !stat->is<IR::EmptyStatement>() */)
```

Comment:
This was an optimization I was considering, but its really independent of the change -- if the converter returns a EmptyStatement, its not really necessary to put it in the action, and a later call to SimplifyControlFlow will remove it.  But if it is meaningless, perhaps its better to never put it in in the first place.